tools/linker/CoreOptimizeGeneratedCode.cs

```diff
@@ -41,18 +41,21 @@ public override bool IsActiveFor (AssemblyDefinition assembly)
 			}
 			
 			// if the assembly does not refer to [CompilerGeneratedAttribute] then there's not much we can do
-			HasGeneratedCode = false;
+			HasOptimizableCode = false;
 			foreach (TypeReference tr in assembly.MainModule.GetTypeReferences ()) {
 				if (tr.Is ("System.Runtime.CompilerServices", "CompilerGeneratedAttribute")) {
 #if DEBUG
 					Console.WriteLine ("Assembly {0} : processing", assembly);
 #endif
-					HasGeneratedCode = true;
+					HasOptimizableCode = true;
```

Comment:
that older condition could be removed
it's even less likely that a new type would be found, so it's a better check without it

Reply:
@spouliot you mean stop checking for `[CompilerGenerated]`, and only check for `[BindingImpl]`? That might impact existing third-party binding projects.